scan/lib/scan/runner.rb

```diff
@@ -10,7 +10,12 @@ def run
     end
 
     def test_app
-      open_simulator_for_device(Scan.device)
+      # We call this method, to be sure that all other simulators are killed
+      # And a correct one is freshly launched. Switching between multiple simulator
+      # in case the user specified multiple targets works with no issues
+      # This way it's okay to just call it for the first simulator we're using for
+      # the first test run
+      open_simulator_for_device(Scan.devices.first)
```

Comment:
@schwmi We've released [`scan` 0.7.1](https://github.com/fastlane/fastlane/releases/tag/scan/0.7.1) with a fix for this. Please let us know if it helps! 👍